public/app/plugins/datasource/cloud-monitoring/components/LabelFilter.tsx

```diff
@@ -27,13 +26,15 @@ const FilterButton = React.forwardRef<HTMLButtonElement, CustomControlProps<stri
 );
 FilterButton.displayName = 'FilterButton';
 
-const OperatorButton = React.forwardRef<HTMLButtonElement, CustomControlProps<string>>(({ value, ...rest }, ref) => {
-  return (
-    <Button {...rest} ref={ref} variant="secondary">
-      <span className="query-segment-operator">{value?.label}</span>
-    </Button>
-  );
-});
+const OperatorButton = React.forwardRef<HTMLButtonElement, CustomControlProps<string>>(
+  ({ value, isOpen, invalid, ...rest }, ref) => {
```

Comment:
Only change here was to add `isOpen, invalid` to the list of destructured props so they're not included in `rest`. This was to get rid of a warning since they are not recognized props on the button.